.github/workflows/build.yml

```diff
@@ -0,0 +1,93 @@
+name: CI
+
+on:
+  pull_request:
+    paths-ignore:
+      - '*.md'
+      - 'LICENSE'
+    branches:
+      - master
+  # This CI will be triigerred on any merge_group events
+  merge_group:
+
+env:
+  RUST_BACKTRACE: 1
+  CARGO_TERM_COLOR: always
+  CARGO_INCREMENTAL: 0
+  CARGO_NET_RETRY: 10
+  CFLAGS_x86_64_fortanix_unknown_sgx: "-isystem/usr/include/x86_64-linux-gnu -mlvi-hardening -mllvm -x86-experimental-lvi-inline-asm-hardening"
+  CC_x86_64_fortanix_unknown_sgx: clang-11
+
+jobs:
+  test:
+    name: Build+Test
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Install additional dependencies
+      run: |
+        sudo apt-key adv --fetch-keys 'https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key'
```

Comment:
Minor security concern around automatically fetching keys in automated systems. It's probably better for us to chek in the current version of this key into the repo and use the checked-in key. All it takes is someone compromising the web site, and then the CI job will just download and install whatever malware the attacker wants.

Reply:
Sure! will do